### PR TITLE
Fix. "TypeError: 'arguments', 'callee', and 'caller' cannot be accessed in this context, when using addTrack() method"

### PR DESCRIPTION
**Pull request by dimitrisniras**
**Description**
PR to fix the issue "TypeError: 'arguments', 'callee', and 'caller' cannot be accessed in this context, when using addTrack() method" #1023

### CODE REVIEW

File: test/unit/safari.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('Safari shim', () => {
 
     it('shimStreamsAPI existence', () => {
       const prototype = window.RTCPeerConnection.prototype;
-      expect(prototype.addTrack.length).to.equal(1);
```
Comment (fippo) [inline test/unit/safari.js]: the point of not adding stream here is that its an optional argument in the IDL, check `RTCPeerConnection.prototype.addTrack.length` without adapter.
That should not be changed.

File: src/js/safari/safari_shim.js [hunk 2]
```diff
@@ -39,14 +39,15 @@ export function shimLocalStreamsAPI(window) {
     };
 
     window.RTCPeerConnection.prototype.addTrack =
-      function addTrack(track) {
-        const stream = arguments[1];
+      function addTrack(track, ...stream) {
```
Comment (fippo) [inline src/js/safari/safari_shim.js]: nit: `... streams` please. See [the spec](https://w3c.github.io/webrtc-pc/#rtcpeerconnection-interface-extensions)
  RTCRtpSender addTrack(MediaStreamTrack track, MediaStream... streams);

**Comment by fippo** [comment]
Thank you! LGTM with a small nit. Can you squash or shall I The electron failure is unrelated, i'll take a look and/or merge regardless.

**Review by fippo — APPROVED** [review]

File: src/js/safari/safari_shim.js (hunk 2 above)
Comment (dimitrisniras) [reply src/js/safari/safari_shim.js]: Fixed that!

**Comment by fippo** [comment]
published as 7.5.1 - sorry for being slow, had planned to fix more thing but... life happened.

**Pull request merged**